### PR TITLE
Fix null in array

### PR DESCRIPTION
**Pull request by e3dio**
fix https://github.com/porsager/postgres/issues/371

### CODE REVIEW

File: src/types.js [hunk 1]
```diff
@@ -243,7 +243,7 @@ export const arraySerializer = function arraySerializer(xs, serializer) {
   if (Array.isArray(first) && !first.type)
     return '{' + xs.map(x => arraySerializer(x, serializer)).join(',') + '}'
 
-  return '{' + xs.map(x =>
+  return '{' + xs.map(x => x == null ? 'null' :
```
Comment (zone117x) [inline src/types.js:246]: Doesn't this library only use strict equality comparisons?

Comment (porsager) [reply src/types.js:246]: Yes, this isn't going in.. I've already got a fix for this in the works, so closing the PR

**Comment by porsager** [comment]
Something is already on the way, but thanks anyway ;)

File: src/types.js (hunk 1 above)
Comment (e3dio) [reply src/types.js:246]: You use whatever equality comparison is needed, this fix works great

**Comment by e3dio** [comment]
Ok could have said something like 5 days ago so I don't need to fix your library for you, thanks

**Pull request closed without merging**